ironfish-cli/src/commands/chain/import.ts

```diff
@@ -87,31 +90,39 @@ export default class ImportSnapshot extends IronfishCommand {
       const bar = CliUx.ux.progress({
         barCompleteChar: '\u2588',
         barIncompleteChar: '\u2591',
-        format: 'Downloading snapshot: [{bar}] {value}% | ETA: {eta}s',
+        format:
+          'Downloading snapshot: [{bar}] {percentage}% | {downloadedSize} / {fileSize} | ETA: {eta}s',
```

Comment:
We can use a meter like rescan, and get a better estimate and render it using renderEstimate()